src/SUMMARY.md

```diff
@@ -32,6 +32,7 @@
     * [New locale](tools/pontoon/adding_new_locale.md).
     * [Notifications](tools/pontoon/sending_notifications.md).
     * [Renaming a localization file](tools/pontoon/renaming_a_file.md).
+    * [Renaming a project](renaming_a_project.md).
```

Comment:
The path is wrong, check the other lines. I would also move it under `New Project`, since they're grouped by topic, not by type of action.
```suggestion
    * [Renaming a project](tools/pontoon/renaming_a_project.md).
```